frontend-new/src/styles/main.css

```diff
@@ -2,17 +2,51 @@
 
 /* TODO do we need this? */
 #nprogress {
-  pointer-events: none;
+    pointer-events: none;
 }
 
 #nprogress .bar {
-  @apply bg-teal-600 opacity-75;
+    @apply bg-teal-600 opacity-75;
 
-  position: fixed;
-  z-index: 1031;
-  top: 0;
-  left: 0;
+    position: fixed;
+    z-index: 1031;
+    top: 0;
+    left: 0;
 
-  width: 100%;
-  height: 2px;
+    width: 100%;
+    height: 2px;
+}
+
+html {
+    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Oxygen', 'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue', sans-serif;
```

Comment:
font is a good question, any preferences? I do like fira sans, but we would need to import it I think

Reply:
I copied it 1:1 from the paper forums, to keep it similar. If we wanna use a font, I personally would vote for a web-safe font, just to a) increase loads speeds and b) not have that flickering font / flash of unstyled content thing